MediaGalleryUi/view/adminhtml/ui_component/standalone_media_gallery_listing.xml

```diff
@@ -23,16 +23,28 @@
             <button name="delete_folder">
                 <param name="on_click" xsi:type="string">jQuery('#delete_folder').trigger('delete_folder');</param>
                 <param name="disabled" xsi:type="string">disabled</param>
-                <param name="sort_order" xsi:type="number">30</param>
+                <param name="sort_order" xsi:type="number">20</param>
                 <class>action-default scalable action-quaternary</class>
                 <label translate="true">Delete Folder</label>
             </button>
             <button name="create_folder">
                 <param name="on_click" xsi:type="string">jQuery('#create_folder').trigger('create_folder');</param>
-                <param name="sort_order" xsi:type="number">40</param>
+                <param name="sort_order" xsi:type="number">30</param>
                 <class>action-default scalable add</class>
                 <label translate="true">Create Folder</label>
             </button>
+	    <button name="delete_selected">
```

Comment:
Hi @Nazar65, it seems this line has an incorrect amount of spaces or tab character
```suggestion
            <button name="delete_selected">
```

Reply:
Yes thanks, need to set correct settings fo xml files :)